examples/games_fps.html

```diff
@@ -69,6 +69,7 @@
 			const container = document.getElementById( 'container' );
 
 			const renderer = new THREE.WebGLRenderer( { antialias: true } );
+			renderer.setAnimationLoop( animate );
```

Comment:
IMHO, it seems odd to insert this optional line prior to `.setPixelRatio()` and `.setSize()`. Personally, I would call this method after all other renderer properties are set.

Reply:
Okay, I have moved the call of `setAnimationLoop()` at the end of all setters. In this way it matched the WebGPU demos and it still looks clean. I mean a fan of mixing the setter calls with the property assignments, tbh.